networking/aws-load-balancer-operator.adoc

```diff
@@ -6,22 +6,33 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+The AWS Load Balancer Operator is an Operator supported by Red{nbsp}Hat that users can optionally install on SRE-managed {product-title} (ROSA) clusters. The AWS Load Balancer Operator manages the lifecycle of the AWS Load Balancer Controller that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in ROSA clusters.
 
-The AWS Load Balancer Operator (ALBO) is an Operator supported by Red{nbsp}Hat that users can optionally install on SRE-managed
-ifndef::openshift-rosa[]
-{product-title}
-endif::openshift-rosa[]
-ifdef::openshift-rosa[]
-{product-title} (ROSA)
-endif::openshift-rosa[]
-clusters. The ALBO manages the lifecycle of the AWS-managed AWS Load Balancer Controller (ALBC) that provisions AWS Elastic Load Balancing v2 (ELBv2) services for applications running in
-ifndef::openshift-rosa[]
-{product-title}
-endif::openshift-rosa[]
-ifdef::openshift-rosa[]
-ROSA
-endif::openshift-rosa[]
-clusters.
+// Creating an AWS IAM role by using the Cloud Credential Operator utility
+include::modules/using-ccoctl-create-iam-role-alb-operator.adoc[leveloffset=+1]
 
+// Creating an AWS IAM role for the controller by using the Cloud Credential Operator utility
+include::modules/using-ccoctl-create-iam-role-alb-controller.adoc[leveloffset=+1]
+
+// Installing an AWS Load Balancer Operator
 include::modules/aws-installing-an-aws-load-balancer-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
```

Comment:
I feel that these add. resources are overshadowing the Uninstalling an AWS Load Balancer Operator. If you (the reviewer) think so as well, I can move the resources after the Uninstalling an AWS Load Balancer Operator section.